CHANGELOG.md

```
@@ -0,0 +1,488 @@
+- Logging
+- Logging trpc error
+- Error handling update
+- Npx shadcn-ui@latest add tooltip
+- Add `@antfu/ni`
+- パフォーマンス調査用
+- Nodeversion の指定に package.json を使う
+- デバッグ用のファイル生成はスクリプトで行う
+- デバッグファイル削除用のコマンド
+- Use bun
+- Use antfu/ni
+- Fix https://github.com/vitejs/vite/issues/15714
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)